person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
- [ ] Please you have a little mistake here. Please write `parent_permission = true` onstead of `parent_permission: true`

Reply:
Hi, @OLIPLICHE thanks for your review.
As you can see, we can't use `=` because it produces linter errors. That's why I won't be able to apply your changes request.

<img width="678" alt="Screen Shot 2022-06-13 at 16 56 19" src="https://user-images.githubusercontent.com/95060298/173452281-5425a0a1-1e35-4067-8809-36d2d4b83311.png">